INSTALLATION.md

```diff
@@ -1,218 +1,156 @@
 
-2. NodeJS installed on your local system(version 14 or above preferrable). It's best to make use of a node version manager to manage nodejs versions on your system. Two popular node version managers currently are [fnm](https://github.com/Schniz/fnm) and [nvm](https://github.com/nvm-sh/nvm). Follow the setup guide for either of them and install NodeJS.
 
-Follow these steps to get the API running :-
+**Note:** Make sure you are in the root directory of the talawa-api repository that you've cloned. The following instructions assume you're already in the root directory.
 
-1.  Clone this repo to your local machine :-
 
-        git clone https://github.com/PalisadoesFoundation/talawa-api
+# Creating .env file
 
-2.  Change directory to cloned folder :-
+A file named `.env` is required in the root directory of talawa-api for storing environment variables required by talawa-api. It is not a part of the repo and you will need to create it. There is a sample file named `.env.sample` in the root directory. Create a new `.env` file by copying the contents of the `.env.sample` file.
 
-        cd talawa-api
+        cp .env.sample .env
 
-3.  Install the npm packages required by talawa-api :-
 
-        npm install
+# Setting up the mongoDB database
 
-4.  Talawa-API uses a configuration file named `.env` in the root directory. It is not a part of the repo and you will need to create it. There is a sample configuration file named `.env.sample` in the root directory. Create a new `.env` file by copying the contents of the `.env.sample` file.
+We're listing a few common approaches to get a running instance of mongodb database:-
 
-        cp .env.sample .env
+1. MongoDB Atlas is the easiest way to get a running instance of mongodb database. It is a hosted(remote) mongodb database provided by mongodb itself. If you're a beginner and don't want too much of a hassle setting up the database you should use this approach. Follow the setup guide on official [mongodb atlas docs](https://www.mongodb.com/docs/atlas/getting-started/). Mongodb Atlas is just one of the many hosted database solutions. If you want to use something else please do your own research.
 
-5.  Generate an `ACCESS_TOKEN_SECRET` using the `openessl` command below and copy the result to the `ACCESS_TOKEN_SECRET` section of the `.env` file.
+2. You can install mongodb natively on your system and create/connect to the database. Follow the setup guide on official [mongodb docs](https://www.mongodb.com/docs/manual/administration/install-community/) for your respective operating system.
 
-        openssl rand -hex 32
+3. If you don't want to bloat your system with unnecessary stuff docker is a great way to manage applications. You can set up a mongodb database inside a docker container and manage it as per your will. Follow the setup guide using this [video tutorial](https://www.youtube.com/watch?v=uklyCSKQ1Po).
```

Comment:
Please mention that there is a docker installation section later in this document.

Reply:
This section and docker section are different. This section includes instructions to set up only the mongodb database using docker. The docker section includes instructions to run talawa-api inside a docker container.